tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md

```diff
@@ -147,35 +153,35 @@ This step will create a catalog that contains a command and an input.  The input
 
     ![input with keys](input.png)
 
-    >Note these credentials will be used by a Cloud Foundry command to return  the details of an SAP HANA database service.  The user should appear under the members list at the space level in the BTP Cockpit, have the role space developer, and not have two-factor authentication enabled.  Additional details on creating a technical user can be found at [Creating a Technical User for Cloud Platform Integration](https://blogs.sap.com/2018/08/17/creating-a-technical-user-for-cloud-platform-integration/).
+    >Note these credentials will be used by a Cloud Foundry command to return the details of an SAP HANA database service.  Ensure the user can login successfully to a site such as [SAP Community](https://community.sap.com/) and not have two-factor authentication enabled.  The user should appear under the members list at the space level in the BTP Cockpit and have the role space developer.  Additional details on creating a technical user can be found at [Creating a Technical User for Cloud Platform Integration](https://blogs.sap.com/2018/08/17/creating-a-technical-user-for-cloud-platform-integration/).
```

Comment:
the user can log in

to log in, the login

Reply:
Will update "Ensure the user can login "
to 
"Ensure the user can log in"